cli/cache/node.rs

```diff
@@ -94,29 +94,6 @@ impl NodeAnalysisCache {
       cjs_analysis,
     ));
   }
-
-  pub fn get_esm_analysis(
-    &self,
-    specifier: &str,
-    expected_source_hash: &str,
-  ) -> Option<Vec<String>> {
-    Self::ensure_ok(
-      self.inner.get_esm_analysis(specifier, expected_source_hash),
-    )
-  }
```

Comment:
Awesome! There is more code above here that can be deleted. Search for "esmglobalscache"